Source/Engine/Core/Types/Variant.h

```diff
@@ -39,6 +39,9 @@ API_STRUCT(InBuild) struct FLAXENGINE_API VariantType
         Vector2,
         Vector3,
         Vector4,
+        Int2,
+        Int3,
+        Int4,
```

Comment:
Also, add support for newly added `VariantTypes` in `FlaxEditor.Utilities.Utils.cs` where C# has interop for Variant.